.travis.yml

```diff
@@ -3,20 +3,26 @@ sudo: required
 before_script:
   - npm install -g bower polymer-cli@next
   - polymer install --variants
+  - sudo mv /usr/bin/google-chrome /usr/bin/google-chrome-old
+  - sudo mv /usr/bin/google-chrome-beta /usr/bin/google-chrome
```

Comment:
Is this really necessary? If its being cached incorrectly, you can manually clear the caches for the repo.